minutes/2022-08-22.md

```diff
@@ -0,0 +1,95 @@
+# Meeting Minutes for August 22, 2022
+
+## Attendees
+
+@fviernau, @MarcelBochtler, @mnonnenmacher, @sschuberth, @tsteenbe
+
+## NOTICE file and copyright change
+
+- New files will be created with the new copyright header, see https://github.com/oss-review-toolkit/ort-governance/issues/6#issuecomment-1173522852.
+- Existing files should not be updated using the new copyright header. This will be done by Sebastian in one big change.
+
+## Marketing & Website
+
+- Thomas is working on an ORT website with @mkurzman and Kerstin Sonnenholzer.
```

Comment:
Again, `@tsteenbe` instead of `Thomas`. And maybe end the sentence with "from Bosch.IO" so "outsiders" know where to put Marcel K. and Kerstin.

Reply:
Added clarifications about Marcel and Kerstin.